sdk/containerorchestratorruntime/Azure.ResourceManager.ContainerOrchestratorRuntime/samples/Generated/Samples/Sample_ConnectedClusterLoadBalancerCollection.cs

```diff
@@ -21,7 +21,7 @@ public partial class Sample_ConnectedClusterLoadBalancerCollection
         public async Task CreateOrUpdate_LoadBalancersCreateOrUpdate()
         {
             // Generated from example definition: 2024-03-01/LoadBalancers_CreateOrUpdate.json
-            // this example is just showing the usage of "LoadBalancer_CreateOrUpdate" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "LoadBalancers_createOrUpdate" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `createOrUpdate`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/kubernetesruntime/KubernetesRuntime.Management/loadbalancer.tsp#L60